project/src/constants.ts

```diff
@@ -12,3 +12,5 @@ export const SORT_TYPE_KEYS = {
   PriceHighToLow: 'PriceHighToLow',
   TopRated: 'TopRated',
 };
+
+export const CITIES_LIST = ['Paris', 'Cologne', 'Brussels', 'Amsterdam', 'Hamburg', 'Dusseldorf'];
```

Comment:
а попробуй ради любопытства добавить слова as const  и навести мышкой на CITIES_LIST
```suggestion
export const CITIES_LIST = ['Paris', 'Cologne', 'Brussels', 'Amsterdam', 'Hamburg', 'Dusseldorf'] as const;
```